lib/rabbit_feed/configuration.rb

```diff
@@ -42,6 +43,7 @@ def connection_options
         options[:heartbeat] = heartbeat if heartbeat
         options[:connect_timeout] = connect_timeout if connect_timeout
         options[:host] = host if host
+        options[:hosts] = hosts if hosts
```

Comment:
sorry,  ask cause I don't know how this works... is this property being picked up automatically by `Bunny`?.. I mean where is this value actually used?